packages/samples/mvp-dapp/src/exception.ts

```diff
@@ -0,0 +1,29 @@
+import { Context, HttpError, Next } from 'koa'
+import { MvpResponse } from './response'
+
+export function exceptionHandle() {
+  return async (ctx: Context, next: Next) => {
+    try {
+      await next()
+    } catch (err) {
+      if (err instanceof HttpError) {
+        ctx.body = { message: err.message }
+      } else if (err instanceof BizError) {
+        ctx.status = 200
+        ctx.body = MvpResponse.err(err.message, err.errCode)
+      } else {
+        ctx.status = 500
+        ctx.body = { message: 'Internal server error' }
+      }
+    }
+  }
+}
+
+export class BizError extends Error {
+  errCode: string
```

Comment:
`code` was used in `MvpResponse`, would it be better to name them in a consistent style?

Reply:
> `code` was used in `MvpResponse`, would it be better to name them in a consistent style?

Okay, I'll refactor it

Reply:
The point I referred is the name style, `code` used in `MvpResponse` while `errCode` is used in `MvpError`

Reply:
> The point I referred to is the name style, `code` used in `MvpResponse` while `errCode` is used in `MvpError`

Oh, I see, the actual `code` and `errCode` are similar